index.html

```diff
@@ -234,7 +234,7 @@ <h2 class="CANPOY">CANPOY</h2>
         <img
           class="about-my-self"
           src="Microverse-Student-Project-1/img6.png"
-          alt=""
+          alt="background-img"
```

Comment:
add some alt information to the image to increase the accessibility for people who don't want to load the images